brush/src/providers/ROMProvider.js

```diff
@@ -1,14 +1,47 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
+import localforage from 'localforage'
 import ROMContext from '../context/ROMContext'
 
+const useROMBuffer = () => {
+  const romBufferStates = {
+    empty: { memory: null, status: 'empty' },
+    loaded: memory => ({ memory, status: 'loaded' }),
+    starting: { memory: null, status: 'starting' },
+  }
+
+  const [romBufferState, setROMBufferState] = useState(romBufferStates.starting)
+
+  useEffect(() => {
```

Comment:
Why not use the `useEffect` with an async function ?
```suggestion
  useEffect( async () => {
```
This could kill the fetchLocalROMBufferMemory

Reply:
I tried to do in this way, but the `useEffect` should returns nothing. Then, when a function is `async`, it'll returns a promise - what I can't do at `useEffect`.